.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: tests
+
+on:
+  pull_request:
+    branches:
+      - "12.0*"
+  push:
+    branches:
+      - "12.0*"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    name: ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo12.0:latest
+            makepot: "true"
+            name: test with Odoo
```

Comment:
I think this is not useful at home, let's save some of Putin's gas for later.

Reply:
OK, will make a few efficiency changes upstream and apply them manually here.